tools/cloudflow-operator/src/main/scala/cloudflow/operator/action/runner/SparkRunner.scala

```diff
@@ -270,12 +261,11 @@ final class SparkRunner(sparkRunnerDefaults: SparkRunnerDefaults) extends Runner
       if (!agentPaths.contains(PrometheusAgentKey)) {
         SparkApp.Monitoring(prometheus = SparkApp.Prometheus(
           jmxExporterJar = "/prometheus/jmx_prometheus_javaagent.jar",
-          configFile = "/etc/cloudflow-runner/prometheus.yaml",
+          configFile = "/prometheus/prometheus.yaml",
           port = 2050))
       } else {
-        SparkApp.Monitoring(prometheus = SparkApp.Prometheus(
-          jmxExporterJar = agentPaths(PrometheusAgentKey),
-          configFile = PrometheusConfig.prometheusConfigPath(Runner.ConfigMapMountPath)))
+        SparkApp.Monitoring(prometheus = SparkApp
+          .Prometheus(jmxExporterJar = agentPaths(PrometheusAgentKey), configFile = "/prometheus/prometheus.yaml"))
```

Comment:
Do we still need agent paths?

Reply:
(It would be great if we could get rid of those)

Reply:
how does this work, pointing to `/prometheus/prometheus.yaml`?

Reply:
I add a comment.